styles.css

```diff
@@ -0,0 +1,43 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+body {
+  font-size: 20px;
+  background-color: rgba(26, 25, 32, 0.991);
+  color: beige;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
```

Comment:
- [ ] @Hazard10-eng , I noticed you haven't completely followed the suggestions of the previous reviewer. As part of this project requirement, You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).  I suggest you pick a template to work with.
- [ ] I would also encourage you to make sure your content is well-aligned with flexbox. This gives us a lot of options to align all our elements, plus, it's responsive. 

> This is how your design looks

![eros](https://user-images.githubusercontent.com/71140133/199247041-fbf53c95-fe87-4902-843f-9c71d28c2136.PNG)